engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -164,6 +165,16 @@ public void processRecord(
         eventKey, ProcessInstanceModificationIntent.MODIFIED, value, command);
   }
 
+  @Override
+  public ProcessingError tryHandleError(final TypedRecord<ProcessInstanceModificationRecord> typedCommand, final Throwable error) {
+    if (error instanceof EventSubscriptionException exception) {
```

Comment:
❓ Is this exception already thrown by this processor? It's not part of this PR

Reply:
Yes, it's already thrown because we use this same logic for start process instance anywhere:
https://github.com/camunda/zeebe/blob/main/engine/src/main/java/io/camunda/zeebe/engine/processing/common/ElementActivationBehavior.java#L245